t/t1092-sparse-checkout-compatibility.sh

```diff
@@ -487,13 +487,14 @@ test_expect_success 'blame with pathspec inside sparse definition' '
 
 # TODO: blame currently does not support blaming files outside of the
 # sparse definition. It complains that the file doesn't exist locally.
-test_expect_failure 'blame with pathspec outside sparse definition' '
+test_expect_success 'blame with pathspec outside sparse definition' '
```

Comment:
Also remove the `TODO` comment.

Reply:
I made the TODO comment because I didn't have time to look into it. I think at this point, this is "by design" and not something we intend to fix, especially because we are making it a `_success` instead of a `_failure`.